tests/Authentication/Filters/SessionFilterTest.php

```diff
@@ -43,7 +44,7 @@ public function testFilterSuccess(): void
         $this->assertSame($user->id, auth('session')->id());
         $this->assertSame($user->id, auth('session')->user()->id);
         // Last Active should have been updated
-        $this->assertNotEmpty(auth('session')->user()->last_active);
+        $this->assertInstanceOf(Time::class, auth('session')->user()->last_active);
```

Comment:
Did rector change this?

Reply:
Yes, 0.18.6.

Reply:
Okay, but in my understanding `chore:` means others like tool config change,
and it does not change the production or test code.
https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716